cocos/animation/marionette/graph-eval.ts

```diff
@@ -472,10 +440,16 @@ class LayerEval {
         return this._currentNode === this._topLevelExit;
     }
 
-    public update (deltaTime: number) {
+    public settle (context: AnimationGraphSettleContext) {
+        for (const state of this._poseStates) {
```

Comment:
`for of` will generate redundant code. just use `for` instead.

Reply:
Solved https://github.com/cocos/cocos-engine/pull/14779/commits/65d0ffcefe67a8537d03e8637868881723b23fa2